src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1813,6 +1815,32 @@ public void testExecuteAndPostProcessRubricQuestion() {
         FeedbackSessionsLogic.inst().deleteFeedbackSessionCascade(fs.getFeedbackSessionName(), fs.getCourseId());
     }
 
+    @Test
+    public void testGetRubricWeights_rubricWeightsForEachCellEmpty_legacyDataWillBeConvertedToNewFormat() {
```

Comment:
>No need to load the data bundle but constructor a FeedbackRubricQuestionDetails should be enough.

@xpdavid Sorry but I did not understand this. Are you suggesting that I should create a constructor in `FeedbackRubricQuestionDetails` that will initialise the `rubricWeights` list?

Reply:
@xpdavid Waiting for your reply on this :sweat_smile:

Reply:
> Are you suggesting that I should create a constructor in FeedbackRubricQuestionDetails that will initialise the rubricWeights list?

No. Don't change production code just for the sake of testing.

Like your method indicated, you are testing `GetRubricWeights()` and that's all. So ideally you should not touch any logic etc (so the test should be inside unit test instead of here). You may want to construct a 1d rubric weight manually (Create a dummy FeedackQuestion) and pass it is to `FeedbackRubricQuestionDetails` to test the method.